styleframe/style_frame.py

```diff
@@ -335,10 +335,23 @@ def row_indexes(self):
 
         return tuple(range(1, len(self) + 2))
 
+    @staticmethod
+    def _to_excel_pandas_defaults(kwargs):
+        """Return five arguments used by `pandas.DataFrame.to_excel()` and
+        use default values in case that arguments not explicit set.
+        """
+        header = kwargs.pop('header', True)
+        startcol = kwargs.pop('startcol', 0)
+        startrow = kwargs.pop('startrow', 0)
+        na_rep = kwargs.pop('na_rep', '')
+
+        return header, startcol, startrow, na_rep
```

Comment:
I suggest to add type hints and to update the docstring. This will require to change line 8 to `from typing import Union, Optional, List, Dict, Tuple, Set, Any`

```suggestion
    @staticmethod
    def _to_excel_pandas_defaults(kwargs: Dict[str, Any]) -> Tuple[bool, int, int, str]:
        """Returns the provided or default values for specific arguments as set by :meth:`pandas.DataFrame.to_excel`
        """

        header = kwargs.pop('header', True)
        startcol = kwargs.pop('startcol', 0)
        startrow = kwargs.pop('startrow', 0)
        na_rep = kwargs.pop('na_rep', '')

        return header, startcol, startrow, na_rep
```